src/js/components/FileInput/utils/formatBytes.js

```diff
@@ -0,0 +1,41 @@
+const SI_CONVERSION_FACTOR = 1000;
+const IEC_CONVERSION_FACTOR = 1024;
+
+const getCurrentOS = () => {
+  const currentOS = ['Win', 'Linux', 'Mac'].find(
+    (v) => window.navigator.userAgent.indexOf(v) >= 0,
```

Comment:
In the HPE Design System site, I'm getting an error when I try to run locally (with latest Grommet stable) of window is not defined. Think we might need to add some fallback for the case that window isn't defined yet?

<img width="999" alt="Screen Shot 2022-11-03 at 1 44 35 PM" src="https://user-images.githubusercontent.com/12522275/199830278-07e870a3-48f3-456b-9fc9-c237b2cc7053.png">

Reply:
Thanks to share it!. I'll work on it 👍🏻

Reply:
Looks like a PR for this has already been filed https://github.com/grommet/grommet/pull/6480 to fix this. Appreciate the willingness to work on this though!